statefun-e2e-tests/statefun-smoke-e2e-java/src/main/java/org/apache/flink/statefun/e2e/smoke/Constants.java

```diff
@@ -0,0 +1,61 @@
+package org.apache.flink.statefun.e2e.smoke;
+
+import org.apache.flink.statefun.e2e.smoke.generated.Command;
+import org.apache.flink.statefun.e2e.smoke.generated.Commands;
+import org.apache.flink.statefun.e2e.smoke.generated.SourceCommand;
+import org.apache.flink.statefun.e2e.smoke.generated.VerificationResult;
+import org.apache.flink.statefun.sdk.java.TypeName;
+import org.apache.flink.statefun.sdk.java.types.SimpleType;
+import org.apache.flink.statefun.sdk.java.types.Type;
+
+final class Constants {
+  private Constants() {}
+
+  private static final String APP_NAMESPACE = "statefun.smoke.e2e";
+  private static final String PROTO_TYPES_NAMESPACE = "type.googleapis.com";
+
+  // =====================================================
+  //  Egresses
+  // =====================================================
+
+  static final TypeName DISCARD_EGRESS = TypeName.typeNameOf(APP_NAMESPACE, "discard-sink");
```

Comment:
I think we should use the driver's Constants.EGRESS_NAME here so that we don't need to worry about the module definition and function definition mismatch problem.

Reply:
Got the point in https://github.com/apache/flink-statefun/pull/248/files#r684575758. That makes sense.